### PR TITLE
translate: guide/schematics (#542)

### PR DESCRIPTION
**Pull request by cleverdog**
## 翻訳・修正

- [x] 変更内容は[CONTRIBUTING.md](https://github.com/angular/angular-ja/blob/master/CONTRIBUTING.md) に記載されたワークフローに従っています

### 備考

Schematics というワードに関しては特に日本語化せず、そのまま利用しています。

### CODE REVIEW

**Comment by cleverdog** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fangular%2Fangular-ja%2Fpull%2F548) for more info**.

@googlebot I signed it!

File: aio-ja/content/guide/schematics.md [hunk 1]
```diff
@@ -1,84 +1,86 @@
-# Generating code using schematics
+# Schematics を使用してコードを生成する
```
Comment (lacolaco) [inline aio-ja/content/guide/schematics.md:1]: タイトルなので名詞形のほうが収まりが良さそうです。
```suggestion
# Schematics を使用したコード生成
```

File: aio-ja/content/guide/schematics.md [hunk 2]
```diff
@@ -1,84 +1,86 @@
...
-These defaults can be overridden by providing a different value for an option on the command line.
-See [Workspace Configuration](guide/workspace-config) for information about how you can change the generation option defaults for your workspace.
+Schematic に関連付けられた JSON スキーマは、コマンドとサブコマンドで使用できるオプションを Angular CLI に通知し、デフォルトを決定します。
+これらのデフォルトは、コマンドラインのオプションに別の値を指定することで上書きできます。
+ワークスペースの生成オプションのデフォルトを変更する方法については、[ワークスペースの設定](guide/workspace-config) を参照してください。
 
-The JSON schemas for the default schematics used by the CLI to generate projects and parts of projects are collected in the package [`@schematics/angular`](https://raw.githubusercontent.com/angular/angular-cli/v7.0.0/packages/schematics/angular/application/schema.json).
-The schema describes the options available to the CLI for each of the `ng generate` sub-commands, as shown in the `--help` output.
+CLI がプロジェクトとプロジェクトの一部を生成するために使用するデフォルトの Schematics の JSON スキーマは、パッケージ [`@schematics/angular`](https://raw.githubusercontent.com/angular/angular-cli/v7.0.0/packages/schematics/angular/application/schema.json) に収集されます。
```
Comment (lacolaco) [inline aio-ja/content/guide/schematics.md:38]: collectではありますがニュアンスとしては集約される、が近いでしょう。

```suggestion
CLI がプロジェクトとプロジェクトの一部を生成するために使用するデフォルトの Schematics の JSON スキーマは、パッケージ [`@schematics/angular`](https://raw.githubusercontent.com/angular/angular-cli/v7.0.0/packages/schematics/angular/application/schema.json) に集約されます。
```

File: aio-ja/content/guide/schematics.md [hunk 3]
```diff
@@ -1,84 +1,86 @@
...
+あなたはこれを、自身の追加 Schematic の例およびモデルとして見ることができます。
 
-Partner and third party libraries also support the Angular CLI with add schematics.
-For example, `@ng-bootstrap/schematics` adds [ng-bootstrap](https://ng-bootstrap.github.io/)  to an app, and  `@clr/angular` installs and sets up [Clarity from VMWare](https://vmware.github.io/clarity/documentation/v1.0/get-started).
+パートナーライブラリとサードパーティライブラリも、Schematics を追加した Angular CLIをサポートしています。
+たとえば、`@ng-bootstrap/schematics` はアプリに [ng-bootstrap](https://ng-bootstrap.github.io/) を追加し、`@clr/angular` は [VMWare から Clarity](https://vmware.github.io/clarity/documentation/v1.0/get-started) をインストールして設定します。
 
-An add schematic can also update a project with configuration changes, add additional dependencies (such as polyfills), or scaffold package-specific initialization code.
-For example, the `@angular/pwa` schematic turns your application into a PWA by adding an app manifest and service worker, and the `@angular/elements`  schematic adds the `document-register-element.js` polyfill and dependencies for Angular Elements.
+また、Schematic を追加すると、設定の変更でプロジェクトを更新したり、依存関係（ポリフィルなど）を追加したり、スキャフォールドパッケージ固有の初期化コードを追加したりできます。
```
Comment (lacolaco) [inline aio-ja/content/guide/schematics.md:66]:
```suggestion
また、追加 Schematicは設定の変更のためにプロジェクトを更新したり、依存関係（ポリフィルなど）を追加したり、スキャフォールドパッケージ固有の初期化コードを追加したりできます。
```

File: aio-ja/content/guide/schematics.md [hunk 4]
```diff
@@ -1,84 +1,86 @@
...
+`add` コマンドはパッケージマネージャーを使用して新しい依存関係をダウンロードし、Schematic として実装されているインストールスクリプトを呼び出します。
 
-For example, the [`@angular/material`](https://material.angular.io/guide/schematics) schematic tells the `add` command to install and set up Angular Material and theming, and register new starter components that can be created with `ng generate`.
-You can look at this one as an example and model for your own add schematic.
+たとえば、[`@angular/material`](https://material.angular.io/guide/schematics) Schematic は、Angular Material とテーマをインストールおよび設定し、`ng generate` で作成できる新しいスターターコンポーネントを登録するように `add` コマンドに指示します。
+あなたはこれを、自身の追加 Schematic の例およびモデルとして見ることができます。
 
-Partner and third party libraries also support the Angular CLI with add schematics.
-For example, `@ng-bootstrap/schematics` adds [ng-bootstrap](https://ng-bootstrap.github.io/)  to an app, and  `@clr/angular` installs and sets up [Clarity from VMWare](https://vmware.github.io/clarity/documentation/v1.0/get-started).
+パートナーライブラリとサードパーティライブラリも、Schematics を追加した Angular CLIをサポートしています。
```
Comment (lacolaco) [inline aio-ja/content/guide/schematics.md:63]:
```suggestion
パートナーライブラリとサードパーティライブラリも、追加 SchematicsによりAngular CLIをサポートしています。
```

Comment (lacolaco) [reply aio-ja/content/guide/schematics.md:63]: `add schematics` がひとつの名詞ですね

**Comment by cleverdog** [comment]
> 翻訳ありがとうございます！コメントした部分の修正をお願いします 🙇
> 
> また、改行位置が原文とずれているので、それも修正お願いします。
> 
> [/CONTRIBUTING.md@`master`#%E6%94%B9%E8%A1%8C%E4%BD%8D%E7%BD%AE%E3%82%92%E5%8E%9F%E6%96%87%E3%81%A8%E6%8F%83%E3%81%88%E3%82%8B](https://github.com/angular/angular-ja/blob/master/CONTRIBUTING.md#%E6%94%B9%E8%A1%8C%E4%BD%8D%E7%BD%AE%E3%82%92%E5%8E%9F%E6%96%87%E3%81%A8%E6%8F%83%E3%81%88%E3%82%8B)

ご確認ありがとうございます、いただいた修正提案を元に修正いたしましたのでご確認よろしくお願いします🙇

**Pull request merged**